portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>My Portfolio Gallery</h1>
+      <div class="Image-thumbnails">
```

Comment:
this is a personal preference thing, but keeping class names as lower cases make them easier to type